config.rb

```diff
@@ -120,6 +120,11 @@ def self.test?
   optional :minio_service_project_id, string
   override :minio_version, "minio_20240406052602.0.0_amd64"
 
+  # VictoriaMetrics
+  optional :victoria_metrics_service_project_id, string
+  override :victoria_metrics_host_name, "victoriametrics.ubicloud.com", string
```

Comment:
Maybe just `metrics.ubicloud.com`? If we eventually replace victoria metrics with something else, it would be good to do in in non-breaking fashion from the dependents perspective.

Reply:
Yes that makes sense, updated